src/constants.ts

```diff
@@ -66,6 +67,7 @@ export const XTK_MANAGEMENT_STAKING_MODULE = 'xtkManagementStakingModule'
 
 // xAssetCLR
 export const AAVE_X_AAVE_A_CLR = 'AAVE-xAAVEa-CLR'
+export const ALPHA_X_ALPHA_A_CLR = 'ALPHA-xALPHAa-CLR'
```

Comment:
Order of the asset<>xAsset is important for xAssetCLR. Please confirm with Hendrick if it's `ALPHA_X_ALPHA_CLR` or `X_ALPHA_A_ALPHA_CLR`